### PR TITLE
[16.0][IMP][web_notify] Add possibility to return an action in a notification (forward port from 10.0)

### PR DESCRIPTION
**Pull request by rousseldenis**
Restore this commit from 10.0:

https://github.com/OCA/web/commit/b333cdc7b1f0b533448597da626a78406be53637#diff-19214d76d5241a7a74edaa364c07c134d4a22ef9d7747046d12600aeaaad97ae

@guewen

### CODE REVIEW

**Review by guewen — APPROVED** [review]

**Comment by rousseldenis** [comment]
Fixed tests

File: web_notify/tests/test_res_users.py [hunk 1]
```diff
@@ -62,7 +82,12 @@ def test_notify_default(self):
         bus_bus = self.env["bus.bus"]
         domain = [("channel", "=", self.env.user.notify_default_channel_name)]
         existing = bus_bus.search(domain)
-        test_msg = {"message": "message", "title": "title", "sticky": True}
+        test_msg = {
+            "message": "message",
+            "title": "title",
+            "sticky": True,
+            "action": None,
```
Comment (IT-Ideas) [inline web_notify/tests/test_res_users.py:89]: Shouldn't we add a test with an action?

**Review by IT-Ideas — COMMENTED** [review]
Thanks!

**Comment by DemchukM** [comment]
In the name of the PR, you need to specify the version [16.0] and add the tag that it is a migration [MIG]
Example: [16.0][MIG]name_module
And in the description briefly describe what was done

**Comment by rvalyi** [comment]
ping @rousseldenis @lmignon @guewen @chienandalu  @IT-Ideas

**Review by guewen — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
:+1:

**Pull request merged**